packages/ui/src/Tooltip/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { RefObject, useRef, FC, useEffect } from "react";
+import { createPortal } from "react-dom";
+
+import { useTooltip } from "./UseTooltip";
+
+type TooltipProperties = {
+  children: React.ReactNode;
+  className?: string;
+  elementRef: RefObject<HTMLElement>;
+  style?: object;
+};
+
+export const TooltipWrapper: FC<TooltipProperties> = ({
```

Comment:
I think we can call it Tooltip instead of TooltipWrapper. What do you say?
```suggestion
export const Tooltip: FC<TooltipProperties> = ({
```